cluster/service.go

```diff
@@ -74,11 +74,21 @@ func (c *Service) Open(ctx context.Context, db schema.Indexer) error {
 		return fmt.Errorf("open raft store: %w", err)
 	}
 
+	var nodeToAddressResolver resolver.NodeToAddress
+	if c.config.EnableFQDNResolver {
+		nodeToAddressResolver = resolver.NewFQDN(resolver.FQDNConfig{
+			// We don't need to specify more config as we are only using that resolver to resolve a node id to an IP
+			// overriding the default resolver using memberlist.
+			TLD: c.config.FQDNResolverTLD,
+		})
+	} else {
+		nodeToAddressResolver = c.config.NodeToAddressResolver
+	}
```

Comment:
**suggestion (if-minor):** maybe we use `c.config.NodeToAddressResolver` the default and replace it by FQDN if it was enabled